compiler/optimizer/Inliner.cpp

```diff
@@ -4689,6 +4694,9 @@ bool TR_InlinerBase::inlineCallTarget2(TR_CallStack * callStack, TR_CallTarget *
    TR::Node * callNode = calltarget->_myCallSite->_callNode;
    TR_VirtualGuardSelection *guard = calltarget->_guard;
 
+   if (getPolicy()->shouldSkipInliningRecognizedMethod(calltarget, tracer()))
```

Comment:
It's probably better to call the new API in `TR_InlinerBase::applyPolicyToTargets` so that the target can be filtered out at the earlier inlining stage. https://github.com/eclipse/omr/blob/add2a6d4a60be70c09f25f3cb590ca68cf9c8574/compiler/optimizer/Inliner.cpp#L4062